src/Filters/FeedbackSort.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Filters;
+
+use DateTime;
+
+class FeedbackSort
+{
+    /**
+     * @var DateTime
+     */
+    public $createdAt = 'desc';
+
+    /**
+     * @var integer
+     */
+    public $stars;
+
+    private function isSortingParamValid($sortingParameter): bool
```

Comment:
This method is used in every `*Sort` class. Extract it to some base abstract class.